src/hotspot/share/runtime/serviceThread.hpp

```diff
@@ -52,7 +52,6 @@ class ServiceThread : public JavaThread {
 
   // Add event to the service thread event queue.
```

Comment:
nit - s/service thread/ServiceThread/
Not your typo, but since you're close by...